tests/integration/test_ncbi_bionlp.py

```diff
@@ -0,0 +1,173 @@
+"""
+This test file tests annotations by using the BioC API (part of bionlp) as
+described in https://www.ncbi.nlm.nih.gov/research/bionlp/APIs/BioC-PMC/
+
+This is an integration test: it will use two API endpoints that can be configured
+by environmental variables:
+- The Nemo-Serve URL, which defaults to https://med-nemo.apps.renci.org/
+- The SAPBERT URL, which defaults to https://med-nemo-sapbert.apps.renci.org/
+"""
+
+import json
+import logging
+import os
+import re
+import urllib.parse
+
+import requests
+import pytest
+
+# Configuration: get the Nemo-Serve URL and figure out the annotate path.
+NEMOSERVE_URL = os.getenv('NEMOSERVE_URL', 'https://med-nemo.apps.renci.org/')
+NEMOSERVE_ANNOTATE_ENDPOINT = urllib.parse.urljoin(NEMOSERVE_URL, '/annotate/')
+NEMOSERVE_MODEL_NAME = "token_classification"
+
+# Configuration: get the SAPBERT URL and figure out the annotate path.
+SAPBERT_URL = os.getenv('SAPBERT_URL', 'https://med-nemo-sapbert.apps.renci.org/')
```

Comment:
do we want babel sapbert here aswell?

Reply:
I was avoiding doing that because there are all kinds of small changes between med-nemo-sapbert and babel-sapbert, but _if you insist_ I've now updated the code to work with Babel Sapbert.